docs/devel/roadmap.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap
+
+The following table contains a priority-ordered list of work items and features the team
+plans to work on (as of March 2016).
+
+P0 items are considered blockers for next (`1.x`) release. All else is tackled on best effort basis.
+
+|Area                         |Item                   |Relative priority                 |
+|-----------------------------|-----------------------|-------------------------|
+|View cluster resources       |Next iteration of UX that scales for all Kubernetes resources and various use cases|P0|
+|View cluster resources       |Handle Services - list and basic details view|P0|
+|View cluster resources       |Handle Replica Sets - list and basic details view|P0|
+|View cluster resources       |Handle Pods - list and basic details view|P0|
+|View cluster resources       |Handle Deployments - list and basic details view|P0|
+|View cluster resources       |Handle Daemon Sets - list and basic details view|P0|
+|View cluster resources       |Handle Pet Sets - list and basic details view|P0|
```

Comment:
Job is more important than PetSet. PetSet probably won't even be enabled by default, and I'm not sure when it will stabilize enough for you to display it in a reasonable way.